File: IAIso-v5.0/core/iaiso-java/iaiso-policy/src/main/java/io/iaiso/policy/CoordinatorConfig.java
package io.iaiso.policy;

/** Coordinator-section configuration from a parsed policy. */
public final class CoordinatorConfig {
    private final double escalationThreshold;
    private final double releaseThreshold;
    private final double notifyCooldownSeconds;

    public CoordinatorConfig(double escalationThreshold, double releaseThreshold,
                             double notifyCooldownSeconds) {
        this.escalationThreshold = escalationThreshold;
        this.releaseThreshold = releaseThreshold;
        this.notifyCooldownSeconds = notifyCooldownSeconds;
    }

    public static CoordinatorConfig defaults() {
        return new CoordinatorConfig(5.0, 8.0, 1.0);
    }

    public double getEscalationThreshold() { return escalationThreshold; }
    public double getReleaseThreshold() { return releaseThreshold; }
    public double getNotifyCooldownSeconds() { return notifyCooldownSeconds; }
}

File: IAIso-v5.0/core/iaiso-php/src/Policy/CoordinatorConfig.php
<?php

declare(strict_types=1);

namespace IAIso\Policy;

/** Coordinator-section configuration from a parsed policy. */
final class CoordinatorConfig
{
    public function __construct(
        public readonly float $escalationThreshold = 5.0,
        public readonly float $releaseThreshold = 8.0,
        public readonly float $notifyCooldownSeconds = 1.0,
    ) {
    }

    public static function defaults(): self
    {
        return new self();
    }
}
